### DISCUSSION TITLE
How To Fix? Character Armour Not Displaying Load Menu.

### DISCUSSION THREAD

**Discussion by WithinAmnesia** [flareteam/flare-game#927] [Q&A] [answered]
@dorkster What do you think about this?
![image](https://user-images.githubusercontent.com/86871191/208320495-e68c53d3-0523-4bae-9526-b9499666b42e.png)
 No armour displayed at load menu?
![image](https://user-images.githubusercontent.com/86871191/208320327-1df22017-34ec-4d64-95cf-796f73430ddd.png) Yet armour is displayed in game?
This is running with 512x512 sprites on the 4K resolution settings. I think I had this bug before but I'm not sure where to find it on github or what it was exactly so I figured to ask.

**Comment by dorkster** [comment]
Not sure what the problem is. Can you share a link to your mod so I can investigate? You can [email it](mailto:jajdorkster@gmail.com) if you don't want to share it publicly.

**Reply by WithinAmnesia** [reply]
I hope to have a 1/3 free shareware demo game eventually like old school Doom. So like also if people like pirate my game I'm not too worried I just want to sell my video game legally when its built properly. If anything I hope for people to like make great games too with Flare and right the wrongs of the past and empower aspiring game developers that they can as one person make great games. Like just picture myself wanting to help people make games basically that's what I'm about lol.
https://mega.nz/file/GYwEwR7I#1zKMaoXl7C2pv20AlMoO4OlZzSqlIob7FbghA4wxQ3g
How to start: Turn on dev tools like on windows 10 I hit f5 and type exec intermap=maps/super_dungeon_prototype.txt then look up AGABAL_Test_Items.txt and spawn those in with f5 exec reward_item=1234 [use the item id where the numbers are] and then also spawn in test enemies / creatures like f5 exec spawn=decayed_skeleton_1,5,5 and then test stuff out and hopefully get more and more content online and feeling great!

**Comment by dorkster** [comment, marked as answer]
Looks like it was an engine bug due to the armor slots being shared across multiple equipment sets: https://github.com/flareteam/flare-engine/commit/17a1244220b4ee2f211a09a58631e78fa16f7ae5

By the way, your mod doesn't work on case-sensitive systems (e.g. anything that's not Windows). My recommendation is to use lowercase for all filenames.

**Reply by WithinAmnesia** [reply]
Okay so I'm working on my game's Alpha Version 0.002. So everything lowercase should help? What else do you think would be good?

**Reply by dorkster** [reply]
>So everything lowercase should help?

Yes. The main issue is that the filename case doesn't match what is in some of your mod files. For example, the items folder is `items/`, but you reference it as `Items/`. Using lowercase across the board just makes managing everything easier.

Besides that, I would try to keep files that aren't part of the what the engine uses out of the mod folder. So that includes Tiled TMX files, source images, notes, etc. Will make it easier when it comes time to actually distribute your mod.

**Reply by WithinAmnesia** [reply]
Okay I followed your recommendation  and I changed all the core feature capital letter to lower case and got the human male and female and the plate armour and the 2 swords wording all in lower case lol. Holy moly that was a lot of renaming and like linking things together lol. Thank you for I caught this thing early on it would be a night mare to fix later on lol. So it very good I had some public testing to find stuff the change early on before making the error potentially like 10,000 times then being like of shit lol. I also added 4k bastard sword loot animations to version 0.002 along side the big lowercase reformatting lol.
